### PR TITLE
feat: must-revalidate and headers enum

### PR DESCRIPTION
**Pull request by ArthurFiorette**
See: #49

### CODE REVIEW

File: test/header/interpreter.test.ts [hunk 1]
```diff
@@ -39,6 +39,15 @@ describe('tests header interpreter', () => {
     expect(result).toBe(10 * 1000);
   });
 
+  it('tests with maxAge=10 and age=7 headers', () => {
```
Comment (Cactusbone) [inline test/header/interpreter.test.ts:42]: it's age=3 :) 7 is the expected ttl

File: src/interceptors/response.ts [hunk 2]
```diff
@@ -87,7 +87,10 @@ export class CacheResponseInterceptor<R, D>
 
     let ttl = response.config.cache?.ttl || this.axios.defaults.cache.ttl;
 
-    if (response.config.cache?.interpretHeader) {
+    if (
+      response.config.cache?.interpretHeader ||
+      this.axios.defaults.cache.interpretHeader
```
Comment (Cactusbone) [inline src/interceptors/response.ts:92]: You're adding the feature to use default axios configuration, this should show in the commit description, to end up in the changelog :)

**Review by Cactusbone — COMMENTED** [review]
LGTM otherwise :)

File: src/interceptors/response.ts (hunk 2 above)
Comment (ArthurFiorette) [reply src/interceptors/response.ts:92]: I wouldn't call it a feature, as I had to be using the default configuration all along.

File: test/header/interpreter.test.ts (hunk 1 above)
Comment (ArthurFiorette) [reply test/header/interpreter.test.ts:42]: nice catch!

**Pull request merged**